packages/core/src/lib/wallet/wallet.types.ts

```diff
@@ -121,13 +121,19 @@ export type InjectedWallet = BaseWallet<
 export type HardwareWalletMetadata = BaseWalletMetadata;
 
 export interface HardwareWalletSignInParams extends SignInParams {
-  derivationPaths: Array<string>;
+  accounts: Array<{
+    derivationPath: string;
+    publicKey: string;
+    accountId: string;
+  }>;
```

Comment:
We should consider extracting this object into a dedicated interface, maybe `HardwareWalletSignInAccount`